custom/icds_reports/reports/prevalence_of_severe.py

```diff
@@ -30,7 +30,7 @@ def get_data_for(filters):
             moderate=Sum('wasting_moderate'),
             severe=Sum('wasting_severe'),
             normal=Sum('wasting_normal'),
-            total=Sum('height_eligible'),
+            total=Sum('nutrition_status_weighed'),
```

Comment:
`nutrtition_status_weighed` should be added to the query, not replace this.

We still need `height_eligible` here, because this is used to figure out how many children are unmeasured (`all_total - measured_total`)

Also  could you change these names to something more obvious? Instead of using `total=Sum('height_eligible')` it would be more readable to have `height_eligible=Sum('height_eligible')` .

I believe similar issue applies for the other changes you made here.